ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -97,41 +96,60 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
       page,
       pageSize,
       force_refresh: true,
+      max_result_set_size: -1,
```

Comment:
what does -1 indicate?

Reply:
Ahh this isn't obvious but bc the cache daemon will truncate responses by default, the `-1` just indicates to still return back all results. I still wanted all results for the `allSessions` call as I need to collect up all the possible targets to filter by and bc this is filtered by user, there shouldn't be that many total sessions.